### DISCUSSION TITLE
Example datasets to use with existing segmentation configs

### DISCUSSION THREAD

**Discussion by furtheraway** [microsoft/InnerEye-DeepLearning#574] [Q&A] [answered]
Dear all,

Besides HelloWords example, I wonder where to find datasets to test use other segmentation configurations like HeadAndNeckPaper.py, Lung.py, and ProstatePaper.py.

The document mentioned "You will find a variety of model configurations here. Those not ending in Base.py reference open-sourced data and can be used as they are." I wonder where can I download those datasets.

Thank you,
John

**Comment by ktakeda1** [comment, marked as answer]
The Lung CT dataset description and download details are in the sample tasks section of documentation here - https://github.com/microsoft/InnerEye-DeepLearning/blob/main/docs/sample_tasks.md

You can use this Head & Neck dataset - https://github.com/deepmind/tcia-ct-scan-dataset

Hope that helps.
